packages/sak-fagsak-profil/src/components/FagsakProfile.tsx

```diff
@@ -30,29 +30,31 @@ const FagsakProfile: FunctionComponent<OwnProps> = ({
   fagsakYtelseType,
   fagsakStatus,
   dekningsgrad,
-  fagsakMarkeringTekst,
+  fagsakMarkeringTekster,
 }) => {
   const intl = useIntl();
   return (
     <>
+      <Heading size="medium">{fagsakYtelseType.navn}</Heading>
+      <VerticalSpacer eightPx />
```

Comment:
VerticalSpacer bør me eigentleg markera som deprecated. Me bør nytta VStack i staden for. Bruker gap for å definera størrelsen på mellomrommet.

Reply:
Lagt inn VStack